res/LICENSE.txt

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
+                           Version 2.0, January 2004
+                        http://www.apache.org/licenses/

```

Comment:
Specify that this is the license for the font by e.g. renaming the license file to `LICENSE-noto-sans.txt`